Justfile

```diff
@@ -220,7 +220,7 @@ package-language-plugins-all: build-language-plugin-bootstrap
 
 # Compiles codegen module
 compile-codegen:
-	scala-cli compile codegen
+	scala-cli --power compile codegen --suppress-experimental-feature-warning
```

Comment:
`--power` shouldn't be needed here

Reply:
it will be in the next PR, when we publish the codegen module ;)